phing/tasks/local-sync.xml

```diff
@@ -17,6 +17,26 @@
 
   <target name="local:sync" description="Synchronize local environment from remote (remote --> local)."
           depends="setup:drupal:settings">
+    <if>
+      <!-- There is multisite config. Allow it to override values for local:sync:site. -->
+      <available file="${blt.config-files.multisite}" />
```

Comment:
Rather than doing this only in the context of local:sync, it seems like we should just check for this file and load its properties for every blt call. We'd add this to property.xml.